devicetypes/qubino/qubino-flush-shutter.src/qubino-flush-shutter.groovy

```diff
@@ -0,0 +1,501 @@
+			}
+		}
+	}
+	events
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd, ep = null) {
+	log.debug "SensorMultilevelReport ${cmd}" + (ep ? " from endpoint $ep" : "")
+	def map = [:]
+	switch (cmd.sensorType) {
+		case 1:
+			map.name = "temperature"
+			def cmdScale = cmd.scale == 1 ? "F" : "C"
+			map.value = convertTemperatureIfNeeded(cmd.scaledSensorValue, cmdScale, cmd.precision)
+			map.unit = getTemperatureScale()
+			break
+		default:
+			map.descriptionText = cmd.toString()
+	}
+	def child = childDevices.find { it.deviceNetworkId == state.temperatureSensorDni }
+	if (!child) {
+		child = createChildDevice("qubinoTemperatureSensor", "Qubino Temperature Sensor", "Qubino Temperature Sensor", 3, "qubino", false)
+		state.temperatureSensorDni = child.deviceNetworkId
+	}
+	child?.sendEvent(map)
+	createEvent(map)
+}
+
+def zwaveEvent(physicalgraph.zwave.Command cmd, ep = null) {
+	log.warn "Unhandled ${cmd}" + (ep ? " from endpoint $ep" : "")
+}
+
+private encap(cmd, endpoint = null) {
+	if (cmd) {
+		if (endpoint) {
+			cmd = zwave.multiChannelV3.multiChannelCmdEncap(destinationEndPoint: endpoint).encapsulate(cmd)
+		}
+		if (zwaveInfo.zw.contains("s")) {
+			zwave.securityV1.securityMessageEncapsulation().encapsulate(cmd).format()
+		} else {
+			cmd.format()
+		}
+	}
+}
+
+private sendEventsToVenetianBlind(events) {
+	if (state.venetianBlindDni) {
+		def child = childDevices.find { it.deviceNetworkId == state.venetianBlindDni }
+		events.each {
+			child.sendEvent(it)
+		}
+		createEvent(descriptionText: "Venetian Blinds level has been updated")
+	} else {
+		log.warn "There's no venetian child device to send events to"
+	}
+}
+
+private createChildDevice(componentName, componentLabel, dthName, childIt, namespace = "smartthings", isComponent = true) {
+	try {
+		def childDni = "${device.deviceNetworkId}:$childIt"
+		def child = addChildDevice(namespace, dthName, childDni, device.getHub().getId(), [
+				completedSetup: true,
+				label         : componentLabel,
+				isComponent   : isComponent,
+				componentName : componentName,
+				componentLabel: componentLabel
+		])
+		return child
+	} catch(Exception e) {
+		log.debug "Exception: ${e}"
+	}
+}
+
+private createVenetianBlindsChildDeviceIfNeeded() {
+	if (state.currentMode.contains("Venetian")) {
+		state.venetianBlindDni = createChildDevice("venetianBlind", "Venetian Blind", "Child Switch Multilevel", 2).deviceNetworkId
+	}
+}
+
+private getParameterMap() {[
+		[
+				name: "Operating modes", key: "operatingModes", type: "enum",
+				parameterNumber: 71, size: 1, defaultValue: 0,
+				values: [
+						0: "Shutter mode",
+						1: "Venetian mode (up/down and slate rotation)"
+				],
+				description: "By setting this parameter to 2 the device enters the calibration mode. The parameter relevant only if the device is set to work in positioning mode (parameter 151 set to 1, 2 or 4)."
```

Comment:
you haven't included the value of 2 as an option so you probably don't want to reference it in the description text?